src/test/java/com/moviecat/test/routers/RPCRouterTest.java

```diff
@@ -0,0 +1,165 @@
+/*
+ *  Licensed to the MovieCat, Inc. under one
+ *  or more contributor license agreements. See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.
+ *
+ *  You may not use this file except in compliance
+ *  with the License.
+ *
+ */
+
+package com.moviecat.test.routers;
+
+import com.moviecat.model.Role;
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@DisplayName("JSONRPC")
+class RPCRouterTest {
+
+    RPCRouter routedMethods;
+    @BeforeEach
```

Comment:
Not sure if it is a good idea to always create a new instance of RPCRouter before each test is run.
`@Before` should be sufficient IMO to create a single re-usable instance for the tests.

Reply:
Great

Reply:
Use `@BeforeAll` instead of `@Before`